crimsobot/models/fun_facts.py

```diff
@@ -0,0 +1,79 @@
+import random
+from typing import Any
+
+from tortoise import fields
+from tortoise.models import Model
+
+from crimsobot.models import DiscordUser
+from crimsobot.models.user import User
+
+
+class NoFactsExist(Exception):
+    def __init__(self, *args: Any):
+        if args:
+            self.message = args[0]
+        else:
+            self.message = None
+
+    def __str__(self) -> str:
+        if self.message:
+            return 'NoFactsExist, {0} '.format(self.message)
+        else:
+            return 'There are no facts about this subject'
+
+
+class FunFacts(Model):
+    uuid = fields.IntField(pk=True)
+    created_by = fields.ForeignKeyField('models.User', related_name='fun_facts', index=True)
+    created_in = fields.BigIntField()
+
+    subject = fields.CharField(max_length=255)
```

Comment:
add `index=True` so we're not doing a table scan trying to find facts

Reply:
i think you missed it in your last commit - guild_id was fixed but subject still has no index